Composer/packages/tools/language-servers/intellisense/tsconfig.build.json

```diff
@@ -0,0 +1,7 @@
+{
+  /* Options used for building production code (tests excluded) */
```

Comment:
Unless something changed, I don't think JSON supports comments.

Reply:
Hmmm. I re-used the same tsconfig.build.json found in every folder.